test/manual/kerberos.test.js

```diff
@@ -65,6 +65,8 @@ describe('Kerberos', function () {
   });
 
   it('validate that gssapiCanonicalizeHostName can be passed in', function (done) {
+    // CNAME resolution on evg macos boxes resolves to a ec2 CNAME which is not associated with the KDC.
+    if (process.platform === 'darwin') this.skip();
```

Comment:
can we do a skipReason for both of the new skips here? also, just for my benefit, does this mean this feature does not apply on macos?